sangbok/main.xml

```diff
@@ -4592,4 +4592,54 @@
 			hur innerligt jag älskar dig!
 		</p>
 	</song>
+	
+	<song name="I Californien"
```

Comment:
Please move this to right after the last song in the "Esoterica" category instead of last in the file, to help with sorting.